lib/utils/forkedstoragetrie.js

```diff
@@ -70,12 +58,14 @@ ForkedStorageTrie.prototype.get = function(key, blockNumber, callback) {
           callback(null, account.serialize());
         });
       } else {
+        if (to.number(blockNumber) > to.number(self.forkBlockNumber)) {
```

Comment:
Reason for change: Whenever we go to the main chain we shouldn't use a block number after we forked because that data is not valid for us.